packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -123,9 +127,10 @@ describe('KeyringController', () => {
 
   it('should restore same vault if old seedWord is used', async () => {
     const currentSeedWord = await keyringController.exportSeedPhrase(password);
+
     const currentState = await keyringController.createNewVaultAndRestore(
       password,
-      currentSeedWord.toString(),
+      currentSeedWord,
```

Comment:
`currentSeedWord` is now exported as a `Uint8Array` and `createNewVaultAndRestore` can receive it in this form (or still as a string as well).

Reply:
> Do you know of any other examples of breaking changes?

🤔 I'll poke around a bit more and see if I can find others!

Reply:
Reading through this I'm not detecting any other breaking changes. But good catch for sure. We will definitely want very thorough QA on both products before these KeyringController updates are pushed to prod.